receiver/skywalkingreceiver/config_test.go

```diff
@@ -0,0 +1,56 @@
+package skywalkingreceiver
+
+import (
+	"github.com/stretchr/testify/assert"
```

Comment:
Imports order needs to be fixed here as well. `make goimports` should help with it

Reply:
ok, thanks

Reply:
You can run `make gofmt`, which should take care of that :)